castai/sdk/api.gen.go

```diff
@@ -51,6 +51,13 @@ const (
 	Ssd     CastaiInventoryV1beta1StorageInfoDeviceType = "ssd"
 )
 
+// Defines values for CastaiSsoV1beta1OIDCType.
+const (
+	TYPEBACKCHANNEL  CastaiSsoV1beta1OIDCType = "TYPE_BACK_CHANNEL"
+	TYPEFRONTCHANNEL CastaiSsoV1beta1OIDCType = "TYPE_FRONT_CHANNEL"
+	TYPEUNKNOWN      CastaiSsoV1beta1OIDCType = "TYPE_UNKNOWN"
+)
+
```

Comment:
[question] why do we have generated file changes in this PR?

Reply:
without this pipeline fails with error "SDK is not up to date, regenerate it by running 'make generate-sdk' and commit your changes."

Reply:
yes, weird. I wasn't able to find any broken merge PR for these files. it could be some dependency change or some 3rd party change triggered this generation.